hypothesis/client
# Bump `frontend-shared` and use (extend) its pattern library

This PR contains three commits:

1. Bump version of `@hypothesis/frontend-shared` to `v1.13.0`
2. Use the pattern-library source from `frontend-shared`
3. Remove unused pattern-library assets (that are now in `frontend-shared`)

## To Test

* Check out this branch
* Run `make dev`
* Visit [the pattern library on your local webserver](http://localhost:3000/ui-playground) and validate that both pattern pages—the "shared" buttons _and_ the "client" buttons—function as expected

Fixes https://github.com/hypothesis/frontend-shared/issues/20
